src/ansys/fluent/core/launcher/fluent_container.py

```diff
@@ -57,7 +57,7 @@ def start_fluent_container(mounted_from: str, mounted_to: str, args: List[str])
                 "-e",
                 "FLUENT_LAUNCHED_FROM_PYFLUENT=1",
                 "ghcr.io/pyansys/pyfluent",
-                "-g",
+                "-gu",
```

Comment:
Apparently this is now necessary, otherwise getting an error while retrieving child-names at `display/set/picture/driver`. Earlier probably it was not going upto that level.